### ISSUE TITLE
OS MasterMap Water Network Layer - extends HY-N

### ISSUE THREAD

**Issue by PeterParslow** [INSPIRE-MIF/2017.2#8]
**Simplified encoding example**

# Short description
OS MasterMap® Water Network Layer offers one of the world’s most detailed, heighted water networks – showing the flow and precise course of every river, stream, lake and canal in Great Britain.

It does this by extending INSPIRE HY-N with one or two extra attributes on each of three features, and a lot on WatercourseLink (& link set)

The key customers were the (England & Wales) Environment Agency, and the (Scottish) Environmental Protection Agency

Specification at: https://www.ordnancesurvey.co.uk/docs/technical-specifications/os-mastermap-water-network-technical-specification.pdf

Schema at: https://www.ordnancesurvey.co.uk/xml/schema/waternetwork/v1/index.html

We achieved this by importing & extending the UML models in EA, and using ShapeChange.

# Example instance
https://www.ordnancesurvey.co.uk/docs/sample-data/os-mastermap-water-network-sample-data.gz#sample-data-download

# Purpose & use
understand water-related risks in detail, combining OS MasterMap Water Layer with OS MasterMap Topography Layer, property information and 3D height data for even richer insights.

For planners, for surveyors, for engineers – this Layer offers the potential to model in 2D and 3D like never before, thanks to the clarity of river flow, gradient and river widths.

Case study: https://www.ordnancesurvey.co.uk/business-and-government/case-studies/dundee-city-council-managing-flood-risk.html

# Used simplification rules
Moved names to links

# Additional information

**Comment by michellutz** [comment]
> ## Used simplification rules
> Moved names to links

What do you mean by this exactly?

**Comment by PeterParslow** [comment]
We extended the WatercourseLink feature type with a couple of 'name' attributes (localName, localNameAlternative). We (& our users) feel that this makes the data simpler - at least, simpler to use for the use cases they have in mind. Of course, if they wanted to produce a graphical output, they would have to de-duplicate the names.

The "alternative" name is a different kind of simplification - it restructures the way to highlight a primary name. In many cases, where we have two names, they are used by two different language communities.

We also simplify the names, by not using the INSPIRE GeographicName type, but the GML 3.3 LocalisedCharacterString, which is more widely implemented (being a use of a basic XML approach).

There are a lot of other extensions in here, which could be considered use case specific, rather than to support simplification (e.g. we've introduced a WatercourseLinkSet feature type, and 'localName' for names on short stretches of a river).
